src/models/wasteWater/story/constants.ts

```diff
@@ -0,0 +1,11 @@
+export const legacyLocations = new Set<string>([
+  'Lausanne (VD)',
+  'Basel (BS)',
+  'Porrentruy (JU)',
+  'Neuchâtel (NE)',
+  'Solothurn (SO)',
+  'Schwyz (SZ)',
+  'test_legacylocation' // for tests
```

Comment:
This should be mocked in the tests to avoid mixing produciton and tests constants.